README.md

```diff
@@ -17,38 +17,51 @@
     $ source env/bin/activate
     ```
 
-3. Install the dependencies
+3. Install the app dependencies
 
     ```
     $ pip install -r requirements/common.txt
     $ pip install -r requirements/dev.txt
     ```
 
-4. Create the database
+4. Other dependencies for running locally
+
+    ```
+    $ gem install foreman
+    $ brew install redis
+    ```
+
+5. Create a .env file in the project root for environment variables
+    
+    To start off, add:
+    ```
+    FLASK_CONFIG=development
+    ```
+
+6. Create the database
 
     ```
     $ python manage.py recreate_db
     ```
 
-5. Other setup (e.g. creating roles in database)
+7. Other setup (e.g. creating roles in database)
 
     ```
     $ python manage.py setup_dev
     ```
 
-6. [Optional] Add fake data to the database
+8. [Optional] Add fake data to the database
 
     ```
     $ python manage.py add_fake_data
     ```
 
-## Running the app
+## Running the app locally
 
 ```
 $ source env/bin/activate
-$ python manage.py runserver
- * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
- * Restarting with stat
+$ redis-server &
+$ foreman start
```

Comment:
Are these commands supposed to be running sequentially? What does foreman do? Only running `foreman start` crashes for me.